ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -179,7 +180,8 @@ private void safelySchedule(TaskRuntime task, OptimizingThread thread) {
     try {
       task.schedule(thread);
     } catch (Throwable throwable) {
-      retryTask(task, false);
+      LOG.error("Schedule task {} failed, put it to retryQueue", task.getTaskId(), throwable);
```

Comment:
Do not use a parameter name like `retryQueue` when printing log.

Reply:
OK, I modified it.